modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/linktoolbar/plugin.js

```diff
@@ -325,7 +325,11 @@
 						link = startElement;
 					}
 
-					const linkInput = instance._toolbar.getItem('LinkInput');
+					const linkInput = instance._toolbar.getItem('LinkInput') ?? config.linkInputValue;
+
+					if (config.target) {
+						instance._toolbar.getItem('LinkTarget').setValue(config.target);
```

Comment:
It isn't changing the value of the select. I'm working now to provide a way to do it. I tried to expose this setValue inside uiselect plugin but it don't work.